src/Microsoft.PowerShell.Commands.Utility/resources/MeasureObjectStrings.resx

```diff
@@ -126,4 +126,7 @@
   <data name="NonNumericInputObject" xml:space="preserve">
     <value>Input object "{0}" is not numeric.</value>
   </data>
+  <data name="AverageSwitchNotSet" xml:space="preserve">
+    <value>StdDeviation was requested and requires the average to be calculated, please add the -Average switch.</value>
```

Comment:
I think we should default `-Average` to be set if `-StdDeviation` is set, I think the error only applies if someone explicitly used `-Average:$false`.  So I think the error message should be a bit more specific:

```none
StdDeviation was requested and requires the average to be calculated, however '-Average' was set to $false.
```